open-reac/src/main/java/com/powsybl/openreac/Reports.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.openreac;
+
+import com.powsybl.commons.reporter.Report;
+import com.powsybl.commons.reporter.Reporter;
+import com.powsybl.commons.reporter.TypedValue;
+
+/**
+ * @author Joris Mancini <joris.mancini_externe at rte-france.com>
+ */
+public final class Reports {
+
+    private Reports() {
+        // Should not be instantiated
+    }
+
+    public static Reporter createVoltageInitReporter(Reporter reporter, String networkId) {
+        return reporter.createSubReporter("voltageInit", "Voltage init on network '${networkId}'", "networkId", networkId);
+    }
+
+    public static void reportConstantQGeneratorsSize(Reporter reporter, int constantQGeneratorsSize) {
+        reporter.report(Report.builder()
+            .withKey("constantQGeneratorsSize")
+            .withDefaultMessage("Reactive power set point is considered fixed for ${size} generators")
```

Comment:
Reactive Target is better to fit with Powsybl wording.

Reply:
Reactive power target or just reactive target ?

Reply:
Reactive power target

Reply:
Ok, fixed.